src/main/webapp/style.css

```diff
@@ -8,10 +8,32 @@ hidden {
 }
 
 .container {
-  margin-top: 0px;
+  margin-top: 10px;
```

Comment:
I think it's better to remove this container style from here because it is going to affect other html files such as the dashboard and the profile page with a margin that may not be wanted. You should use bootstrap spacing utilities and margin classes: https://getbootstrap.com/docs/4.0/utilities/spacing/